medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -0,0 +1,877 @@
+# coding=utf-8
+# Author: p0psicles
+#
+# This file is part of SickRage.
+#
+# SickRage is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SickRage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with SickRage. If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import cgi
+import getpass
+import logging
+import os
+import re
+import tempfile
+import time
+import warnings
+from six import next, iteritems
```

Comment:
hmm okay, how should I resolve this? import it like: from six import next as next_six? As i'd like to use a python2/3 compatible method.